k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -168,6 +168,7 @@
     private static boolean mAnimations = true;
 
     private static boolean mConfirmDelete = false;
+    private static boolean mConfirmMenuDiscard = true;
```

Comment:
The field name doesn't really capture what this option does. I think `mConfirmDiscardMessage` would be a better choice.